animation_nodes/nodes/object/collection_operations.py

```diff
@@ -0,0 +1,33 @@
+import bpy
+from ... base_types import AnimationNode, VectorizedSocket
+
+class CollectionOperationsNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_CollectionOperationsNode"
+    bl_label = "Collection Operations"
+    codeEffects = [VectorizedSocket.CodeEffect]
+
+    useObjectList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput("Collection", "Collection", "collection", defaultDrawType = "PROPERTY_ONLY")
+        self.newInput(VectorizedSocket("Object", "useObjectList",
+            ("Object", "object", dict(defaultDrawType = "PROPERTY_ONLY")),
+            ("Objects", "objects", dict(defaultDrawType = "PROPERTY_ONLY"))))
+            
+        self.newInput("Boolean", "Linked", "linked")
+        self.newOutput("Collection", "Collection", "collection")
+
+    def getExecutionCode(self, required):
+        return "self.executeSingleObject(collection, object, linked)"
```

Comment:
You forgot to assign the return, it should be `collection = self.self.executeSingleObject(...)`. Also, please add a new line at the end of the file, you can setup your editor to add this automatically.